pkg/users/account-details.js

```diff
@@ -96,40 +96,40 @@ function get_expire(name) {
             .then(parse_expire);
 }
 
-export function AccountDetails({ accounts, groups, current_user, user, shells }) {
+export function AccountDetails({ account, details, groups, isLoading, current_user, user, shells }) {
     const [expiration, setExpiration] = useState(null);
 
     useEffect(() => {
         get_expire(user).then(setExpiration);
-    }, [user, accounts]);
+    }, [user, details]);
 
-    const [edited_real_name, set_edited_real_name] = useState(null);
-    const [committing_real_name, set_committing_real_name] = useState(false);
+    const [editedRealName, setEditedRealName] = useState(null);
+    const [comittingRealName, setCommittingRealName] = useState(false);
 
-    const [edited_locked, set_edited_locked] = useState(null);
+    const [editedLocked, setEditedLocked] = useState(null);
 
-    function change_real_name() {
-        if (edited_real_name === null || edited_real_name === undefined)
+    function changeRealName() {
+        if (editedRealName === null || editedRealName === undefined)
             return;
 
-        set_committing_real_name(true);
+        setCommittingRealName(true);
 
         // TODO: unwanted chars check
-        cockpit.spawn(["/usr/sbin/usermod", user, "--comment", edited_real_name],
+        cockpit.spawn(["/usr/sbin/usermod", user, "--comment", editedRealName],
                       { superuser: "try", err: "message" })
                 .then(() => {
-                    set_edited_real_name(null);
-                    set_committing_real_name(false);
+                    setEditedRealName(null);
+                    setCommittingRealName(false);
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-21865-e5f93f63-20250722-133313-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)